src/views/Home/components/About/index.scss

```diff
@@ -0,0 +1,35 @@
+@use "./stylesheets/responsiveness";
+
+.about-component {
+    background: linear-gradient(#8296D9,#BFACF0, #8F9ADD, #D99CE3);
+    position: relative;
+    z-index:-1;
```

Comment:
```suggestion
    z-index: 1;
```
Also, this shouldn't be -1. It's messing up the cursor for text selection on chrome.

Reply:
This can be 0 as well, since `&__background` is at -1.